src/QMCDrivers/DMC/WalkerControl.cpp

```diff
@@ -236,10 +225,16 @@ void WalkerControl::branch(int iter, MCPopulation& pop, bool do_not_branch)
     throw std::runtime_error("Potential bug! Population num_global_walkers mismatched!");
 #endif
 
+  // If we are in warmup it think it should be a fatal error is walker->Multiplicity != 1.0
```

Comment:
DMC warmup follow normal branching. Only the first iteration flags do_not_branch.

Reply:
Remove this comment?